website/static/js/atjsConfig.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * At.js config.
+ */
+'use strict';
+
+var $ = require('jquery');
+var ko = require('knockout');
+
+require('Caret.js');
+require('At.js');
+
+// preventing unexpected contenteditable behavior
+var onPaste = function(e){
+    e.preventDefault();
+    var pasteText = e.originalEvent.clipboardData.getData('text/plain');
+    document.execCommand('insertHTML', false, pasteText);
+};
+
+// remove br if no text
+var onlyElementBr = function() {
+    if (this.innerText.trim() === '') {
+        this.innerHTML = '';
+    }
+};
+
+// make sure br is always the lastChild of contenteditable so return works properly
```

Comment:
What does return do if br is not the lastChild of contenteditable? Is that where we start seeing extra `<p>` tags and such?